examples/diagnostics.ipynb

```diff
@@ -605,14 +605,14 @@
     }
    ],
    "source": [
-    "from devinterp.slt.mala import MalaAcceptanceRate\n",
+    "from devinterp.backends.default.slt.mala import MalaAcceptanceRate\n",
```

Comment:
I think we should make the backends a private module, i.e., `_backends`, so that users import directly from the straightforward thing